src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseInstructionDetailsByCrnTests.kt

```diff
@@ -0,0 +1,47 @@
+package uk.gov.justice.digital.hmpps.hmppsallocations.integration.unallocatedcases
+
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsallocations.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsallocations.integration.mockserver.WorkforceAllocationsToDeliusApiExtension.Companion.workforceAllocationsToDelius
+
+class GetCaseInstructionDetailsByCrnTests : IntegrationTestBase() {
+
+  @Test
+  fun `can get case instruction details by crn and convictionNumber`() {
```

Comment:
Might be worth having a test for when there is no staff data.. I know it's unlikely but so is having an invalid crn...

Reply:
I guess if there is no staff data it should error? we wouldn't want to allocation to go through for staff not found